src/routing_table/mod.rs

```diff
@@ -300,9 +301,10 @@ impl<T: Binary + Clone + Copy + Debug + Default + Hash + Xorable> RoutingTable<T
 
     pub fn is_in_our_group(&self, name: &T) -> bool {
         if self.our_group_prefix.matches(name) {
-            return unwrap!(self.groups.get(&self.our_group_prefix)).contains(name);
+            unwrap!(self.groups.get(&self.our_group_prefix)).contains(name)
```

Comment:
@Fraser999 I agree about early return when there's much code coming afterwards, but here it seemed simpler. A bit pointless change really though :/